src/openms/source/CHEMISTRY/ElementDB.cpp

```diff
@@ -139,201 +139,430 @@ namespace OpenMS
+    buildElement_("Thallium", "Tl", (unsigned int) 81, thallium_abundance, thallium_mass);
+
+
+    map<unsigned int, double> lead_abundance = {{(unsigned int) 204, 0.013999999999999999}, {(unsigned int) 206, 0.24100000000000002}, {(unsigned int) 207, 0.221}, {(unsigned int) 208, 0.524}, };
+    map<unsigned int, double> lead_mass = {{(unsigned int) 204, 203.973043600000011}, {(unsigned int) 206, 205.974465299999991}, {(unsigned int) 207, 206.975896900000009}, {(unsigned int) 208, 207.976653800000008}};
+    buildElement_("Lead", "Pb", (unsigned int) 82, lead_abundance, lead_mass);
+
+
+    map<unsigned int, double> bismuth_abundance = {{(unsigned int) 209, 1.0}, };
+    map<unsigned int, double> bismuth_mass = {{(unsigned int) 209, 208.980398699999995}};
+    buildElement_("Bismuth", "Bi", (unsigned int) 83, bismuth_abundance, bismuth_mass);
+
+
+    map<unsigned int, double> rhenium_abundance = {{(unsigned int) 185, 0.374}, {(unsigned int) 187, 0.626}, };
+    map<unsigned int, double> rhenium_mass = {{(unsigned int) 185, 184.952955000000003}, {(unsigned int) 187, 186.95575310000001}};
+    buildElement_("Rhenium", "Re", (unsigned int) 75, rhenium_abundance, rhenium_mass);
+
+
+    map<unsigned int, double> neodymium_abundance = {{(unsigned int) 142, 0.272}, {(unsigned int) 143, 0.122}, {(unsigned int) 144, 0.23800000000000002}, {(unsigned int) 145, 0.083}, {(unsigned int) 146, 0.172}, {(unsigned int) 148, 0.057999999999999996}, {(unsigned int) 150, 0.055999999999999994}, };
+    map<unsigned int, double> neodymium_mass = {{(unsigned int) 142, 141.907723299999987}, {(unsigned int) 143, 142.909814299999994}, {(unsigned int) 144, 143.910087299999987}, {(unsigned int) 145, 144.912573600000002}, {(unsigned int) 146, 145.913116900000006}, {(unsigned int) 148, 147.916892999999988}, {(unsigned int) 150, 149.920891000000012}};
+    buildElement_("Neodymium", "Nd", (unsigned int) 60, neodymium_abundance, neodymium_mass);
+
+
+    map<unsigned int, double> thorium_abundance = {{(unsigned int) 230, 0.0002}, {(unsigned int) 232, 0.9998}, };
+    map<unsigned int, double> thorium_mass = {{(unsigned int) 230, 230.033133800000002}, {(unsigned int) 232, 232.038055299999996}};
+    buildElement_("Thorium", "Th", (unsigned int) 90, thorium_abundance, thorium_mass);
+
+
+    map<unsigned int, double> lanthanum_abundance = {{(unsigned int) 138, 0.00089}, {(unsigned int) 139, 0.99911}, };
+    map<unsigned int, double> lanthanum_mass = {{(unsigned int) 138, 137.907112000000012}, {(unsigned int) 139, 138.906353300000006}};
+    buildElement_("Lanthanum", "La", (unsigned int) 57, lanthanum_abundance, lanthanum_mass);
+
+
+    map<unsigned int, double> samarium_abundance = {{(unsigned int) 144, 0.0308}, {(unsigned int) 147, 0.15}, {(unsigned int) 148, 0.1125}, {(unsigned int) 149, 0.1382}, {(unsigned int) 150, 0.0737}, {(unsigned int) 152, 0.26739999999999997}, {(unsigned int) 154, 0.2274}, };
```

Comment:
I don't recall which but I remember that a user requested an element.
Did you add that one?

Reply:
Yes, the missing 76 isotope of Germanium is already added.